NameGame/NameGame/APIService.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  APIService.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/12/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+/**
+    Reads and models data from online API.
+ */
+
+class APIService {
+    
+    static let manager = APIService()
+    
+    // Url for user records API
+    let apiString: String = "https://willowtreeapps.com/api/v1.0/profiles/"
+    
+    // Load data from JSON file into a list of UserRecord objects.
+    func loadData(completion: @escaping ([UserRecord]) -> Void) {
```

Comment:
Let's make this function name a bit more explicit. How about changing it to `fetchUserRecords`?